bin/u-boot-test-console

```diff
@@ -32,4 +32,9 @@ if [ -z "${console_baud}" ]; then
     console_baud=115200
 fi
 
+# Force a reset into FEL mode
+if [ "${recovery_impl}" == "sunxi-fel" ]; then
```

Comment:
So, I've dropped this part locally since it's not reliable enough.  The problem here isn't sunxi specific, but rather board design specific and commonly seen on sunxi designs (I encountered this first elsewhere even).  The problem is that due to layout issues enough power can leak over the UART to keep the CPU on even when main power is turned off.  It's not enough power for the system to be at all stable but it is enough to keep it from going off.  So the above is a hack to tell U-Boot to reset the board which in turn drops is into the ROM being in FEL mode and thus when we power the board up again for the next tests it's in the correct state.

I've replaced this locally by just saying that my "ensure everything got powered off" script also tells U-Boot on this board to reset.